nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestTailFile.java

```diff
@@ -311,6 +313,79 @@ private File rollover(final int index) throws IOException {
     }
 
 
+    @Test
+    public void testFileWrittenToAfterRollover() throws IOException, InterruptedException {
+        Assume.assumeTrue("Test requires renaming a file while a file handle is still open to it, so it won't run on Windows", !SystemUtils.IS_OS_WINDOWS);
+
+        runner.setProperty(TailFile.ROLLING_FILENAME_PATTERN, "log.*");
+        runner.setProperty(TailFile.START_POSITION, TailFile.START_BEGINNING_OF_TIME.getValue());
+        runner.setProperty(TailFile.REREAD_ON_NUL, "true");
+        runner.setProperty(TailFile.ROLLOVER_TAIL_PERIOD, "10 mins");
+
+        // first line fully written, second partially
```

Comment:
I'd rather remove these comments. Code is clear enough and some of these already seem to be outdated.
Like this one: why do we consider the second line partial?